tests/lib/rules/require-prop-comment.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * @author *****your name*****
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/require-prop-comment')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('require-prop-comment', rule, {
+  valid: [
+    {
+      code: `
+      <template>
+        <div>1</div>
+      </template>
+      <script>
+      import { defineComponent } from '@vue/composition-api'
+
+      export default defineComponent({
+        props: {
+          /**
+           * a comment
+           */
+          a: Number,
+          // b comment
+          b: Number,
+          // c
+          // comment
+          c: Number
+        }
+      })
+      </script>
+      `
+    },
+    {
+      code: `
+      <template>
+        <div>1</div>
+      </template>
+      <script setup>
+      const props = defineProps({
+        /**
+         * a comment
+         */
+        a: Number
+      })
+      </script>
+      `
+    },
+    {
+      code: `
+      import { defineComponent } from '@vue/composition-api'
+      export const ComponentName = defineComponent({
+        props: {
+          /**
+           * a comment
+           */
+          a: Number
+        },
+        render() {
+          return <div>1</div>
```

Comment:
JSX requires additional test setup and thus errors (see the CI failures). Since it is not needed in the test cases for this rule, please just remove it.